src/zjs_sensor_gyro.json

```diff
@@ -2,7 +2,7 @@
     "module": "sensor_gyro",
     "constructor": "Gyroscope",
     "depends": ["sensor"],
-    "targets": ["arduino_101"],
+    "targets": ["arduino_101", "frdm_k64f"],
```

Comment:
Not sure this makes sense if as per IRC it's not actually going to work on k64f.